tests/client/conftest.py

```diff
@@ -69,7 +69,7 @@ def is_using_real_services(request):
 
 @pytest.fixture(scope='session')
 def client():
-    return ScrapinghubClient(auth=TEST_ADMIN_AUTH,
+    return ScrapinghubClient(apikey=TEST_ADMIN_AUTH,
```

Comment:
@chekunkov hm, that's true, but Dash doesn't support authorisation via `job_key:JWT_token`, are we going to add it for consistency? or you propose to leave it up to user, but put a list with jobkey-auth supported logic somewhere to the docs? if latter, should we check for it and print a warning when instantiating Connection?